Packs/FeedCyjax/IndicatorFields/indicatorfield-Industry_types.json

```
@@ -0,0 +1,26 @@
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "industrytypes",
```

Please change it to:
```suggestion
    "cliName": "cyjaxindustrytypes",
```